src/dmn/trn_transitd.h

```diff
@@ -69,7 +69,6 @@
 		e.key = itf;                                                   \
 		ep = hsearch(e, FIND);                                         \
 		if (ep) {                                                      \
-			free(ep->key);                                         \
```

Comment:
I don't fully understand why this causes the table to return a value that alternate between ep->data == NULL and ep->data is not NULL. What am I missing?